features/manage-position/Create.tsx

```diff
@@ -98,6 +101,7 @@ const Create = () => {
   };
 
   if (
+    network !== null &&
```

Comment:
ooc why this extra check?

Reply:
we need to read `network` later to fetch EMPS[network.id]